opencog/reasoning/pln/rules/deduction.scm

```diff
@@ -1,102 +0,0 @@
-; =============================================================================
-; Deduction Rule
-; TODO The rule must be applicable to ImplicationLink, SubsetLink and PartOfLink
-;
-; AND(Inheritance A B, Inheritance B C) entails Inheritance A C
-; -----------------------------------------------------------------------------
-(load "formulas.scm")
-(define deduction-rule
-    (BindLink
-        (VariableList
-            (VariableNode "$A")
-            (VariableNode "$B")
-            (VariableNode "$C"))
-        (AndLink
-            (InheritanceLink
-                (VariableNode "$A")
-                (VariableNode "$B")
-            )
-            (InheritanceLink
-                (VariableNode "$B")
-                (VariableNode "$C")
-            )
-            ; To avoid matching (Inheritance A B) and (Inheritance B A)
-            (NotLink
```

Comment:
Hmm, and PLNRuleUTest didn't catch it, I understand why, it doesn't produce invalid outputs, just useless...